taxi/migrations/0002_alter_car_options_alter_driver_options_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 5.0.1 on 2024-01-30 09:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("taxi", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="car",
+            options={"verbose_name": "car", "verbose_name_plural": "cars"},
+        ),
+        migrations.AlterModelOptions(
+            name="driver",
+            options={"verbose_name": "driver", "verbose_name_plural": "drivers"},
+        ),
+        migrations.AlterModelOptions(
+            name="manufacturer",
+            options={
+                "verbose_name": "manufacturer",
+                "verbose_name_plural": "manufacturers",
+            },
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app per pull request in the Django projects. Include one migration to not overload the `migrations` folder.